src/main/resources/locales/en/subscription-management.json

```diff
@@ -8,6 +8,8 @@
     "header3": "Subscriptions by court or tribunal ("
   },
   "noSubs": "You currently have no subscriptions",
+  "noSubsCase": "You currently have no subscriptions by cases",
```

Comment:
Cases should be singular according to the screenshot in the ticket

Reply:
changed